src/review_gator/review_gator.py

```diff
@@ -452,6 +462,16 @@ def get_mps(repo, branch, output_directory=None):
                 if comment is not None:
                     result = comment.vote
                     review_date = comment.date_created
+
+                    src_git_repo = mp.source_git_repository_link.replace(
+                        'https://api.launchpad.net/devel/',
+                        'https://git.launchpad.net/')
+
+                    tmpdir = tempfile.TemporaryDirectory()
+                    cloned_repo = get_git_repo(src_git_repo, mp.source_git_path.replace('refs/heads/', ''), tmpdir)
+                    cloned_head_date = cloned_repo.head.commit.committed_datetime
+                    if review_date < cloned_head_date and result.lower() in ['approve', 'approved']:
```

Comment:
I think "needs fixing" and maybe a "needs information" status would also make sense to receive a -STALE suffix.

Reply:
Makes sense. I've added a commit that removes the requirement that the review type be an Approval. This way, any review string will be appended with `-STALE` if the review is older than HEAD. However, only stale approvals will change color. Let me know if that works for you.

Reply:
That sounds good to me, thanks!